src/main/java/hlf/java/rest/client/config/KafkaProperties.java

```diff
@@ -17,28 +20,25 @@
 @RefreshScope
 public class KafkaProperties {
 
-  private ConsumerProperties integration;
+  private List<Consumer> integrationPoints;
   private Producer eventListener;
```

Comment:
[nitpick] the name `eventListener` is in the context of blockchain events. It is confusing otherwise for readability.

Reply:
There ideally would be multiple such listeners from blockchain, can be fixed in later PRs.